src/components/AddItems.jsx

```diff
@@ -27,16 +27,23 @@ export function AddItems() {
 			const daysUntilNextPurchase =
 				event.target.elements['purchase-date'].value;
 
-			if (itemName === '') {
-				alert('Please add an item name.');
-				return;
-			}
-			if (!daysUntilNextPurchase) {
-				alert('Please select an option for date');
-				return;
-			}
-
 			try {
+				const normalizedItemName = itemName
+					.trim()
+					.toLowerCase()
+					.replace(/[&\/\\#, +$!,~%.'":*?<>{}]/g, '');
```

Comment:
Since we use these operations to normalize the name, it would make sense to abstract out this logic into it's on function, so that the same transformation can be applied to all names